src/Orleans.Streaming.Abstractions/Orleans.Streaming.Abstractions.csproj

```diff
@@ -7,5 +7,6 @@
     <TargetFrameworks>$(StandardTargetFrameworks)</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <RootNamespace>Orleans</RootNamespace>
+    <ProduceReferenceAssembly>false</ProduceReferenceAssembly>
```

Comment:
Is this a workaround for some issue where the ref assemblies cause bad interactions with codegen?

Reply:
Yes, this is something that will likely need deeper investigation. The Source Generator only has ref assemblies available to it, and they do not include private members, so when codegen tries to generate a serializer for a type whose base is in this assembly (StreamIdentity), it complains that it might be missing some fields.

There are a few approaches to fixing it:
* Delegate serialization of base type fields like Hagar does
* Migrate to a different serializer (eg, Hagar)
* Change these certain types so that they are interfaces instead
* Make exceptions for these known-empty abstract base types